docs/custom-setups/custom_token.rst

```diff
@@ -18,20 +18,22 @@ In order to deploy the Custom Token, you will need to:
 Get Goerli Testnet ETH (GÖETH)
 ==============================
 
-Goerli is the name of the testnet on Ethereum where we will be deploying our token. 
+Goerli is the name of the testnet on Ethereum where we will be deploying our token.
 To pay for the on-chain transactions you'll first need to get some Goerli ETH (GÖETH) on your Ethereum account.
 
 Since we will be using the Goerli testnet you'll have to switch from **"Main Ethereum Network"** to **"Goerli Test Network"** in your MetaMask by selecting it from the top right dropdown.
 
-After switching to the Goerli testnet you can go ahead and acquire some Goerli testnet ETH by following these steps:
+After switching to the Goerli testnet you can go ahead and acquire some Goerli testnet ETH.
+One option to do so is to follow these steps:
 
-1. Copy your address. It is located on the left hand side in MetaMask and starts with "0x".
-2. Visit the `Goerli faucet <https://goerli-faucet.slock.it/>`_, there you'll have to paste the address into the input field and check the *"I'm not a robot"*  checkbox.
-3. Click on **"REQUEST 0.05 GÖETH"**.
+1) Visit `app.mycrypto.com <https://app.mycrypto.com>`_
+2) When asked how to access your assets, choose MetaMask and connect your account
+3) On the Dashboard, go to **Tools -> Use Testnet Faucet**
+4) Choose your account from the list and **Request Access**
 
-You will see a success message when the request has gone through. If it fails, keep trying until it succeeds.
+You will see a success message showing the corresponding transaction on the Goerli testnet.
 
-You can verify that 0.05 GÖETH was actually received by opening MetaMask and see if your Goerli Test Network shows a balance of total 0.05 GÖETH.
+You can verify that 0.01 GÖETH was actually received by opening MetaMask and see if your Goerli Test Network shows a balance of total 0.05 GÖETH.
```

Comment:
should that second number be 0.01 as well?

Reply:
you're right, updated